README.md

```diff
@@ -45,14 +45,20 @@ FHIR® is the registered trademark of HL7 and is used with the permission of HL7
 Use the .Net Core 3.1 SDK to build FHIR Tools for Anonymization. If you don't have .Net Core 3.1 installed, instructions and download links are available [here](https://dotnet.microsoft.com/download/dotnet-core/3.1).
 
 ## Get sample FHIR files
-This repo contains a few [sample](samples/fhir-r4-files) FHIR files that you can download. These files were generated using  [Synthea&trade; Patient Generator](https://github.com/synthetichealth/synthea). 

```

Comment:
Please make sure the selected STU 3 sample files work with our commandline tool.